static/docs/user-guide/cache-file-linking.md

```diff
@@ -0,0 +1,93 @@
+# File Link Types for the DVC Cache
```

Comment:
I'm thinking if we should rename it to more user friendlier version - "Performance with large data" or something similar? Or are you thinking to put together a separate sections for that?

Reply:
Yes! Actually this is not even the main new guide you requested in #265 **add a user guide page about the optimization (advanced mode)**. My plan was to add another guide for that matter, which will link back to this one.

Maybe I've added too much optimization info already to this one guide so no longer sure what will the optimization guide will contain. Is there anything else to optimize besides cache file link types?

Reply:
no, not that much .. I think this guide alone is probably enough. I would rename, may be write a brief motivation - that space and speed are important and we can't just copy, etc, etc. and call it a section :)